### ISSUE TITLE
Issue with pintool integration on Ubuntu 18.04

### ISSUE THREAD

**Issue by svalat** [memtt/numaprof#9]
Get compilation error coming from pintool plugin building : 

```sh
/home/svalat/Downloads/numaprof/src/integration/pintool/numaprof.cpp: In function ‘int main(int, char**)’:
/home/svalat/Downloads/numaprof/src/integration/pintool/numaprof.cpp:1067:41: error: ‘new’ of type ‘numaprof::ProcessTracker’ with extended alignment 64 [-Werror=aligned-new=]
  gblProcessTracker = new ProcessTracker();
```

**Comment by svalat** [comment]
Humm, still an issue, now not at build time but at loading time : 

```
E: Unable to load /home/sebv/testinstall/lib/libnumaprof-pintool.so: dlopen failed: cannot locate symbol "_ZnwmSt11align_val_t" referenced by "/home/sebv/testinstall/lib/libnumaprof-pintool.so"...
```

**Comment by computereasy** [comment]
I have similar issue on Ubuntu 18.04. Any idea on how to solve it?

**Comment by svalat** [comment]
Still have the issue, will look on this when will have a little bit of time, will go faster as someone need it. Thanks for reporting.

**Comment by svalat** [comment]
Hum it is what I thinked it comes from Pin Mutex class. Make back issue at compile time. Still need to find a work arround. Maybe need to send a mail to pin team.

**Comment by Dr-Noob** [comment]
Maybe someone could clarify that this was already solved in 8ad2eb19ce49be7cb1ca847670c279391dd660d4

**Comment by svalat** [comment]
Fixed by using last version of pintool
